.github/workflows/milestone-checker.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow checks that there is either a 'pr/no-milestone' label applied
+# to a PR or there is a milestone associated with a PR
+
+name: Check Milestone
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled]
+    # Runs on PRs to main and release branches
+    branches:
+      - main
+      - release/**
+
+jobs:
+  # checks that a milestone entry is present for a PR
+  milestone-check:
+    # If there  a `pr/no-milestone` label we ignore this check
+    if: "!contains(github.event.pull_request.labels.*.name, 'pr/no-milestone')"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v2
+        with:
+          repository: "grafana/grafana-github-actions"
+          path: ./actions
+          ref: main
```

Comment:
Can we lock this to a ref to prevent supply chains attack in the grafana github actions?